scripts/build/gen_syscalls.py

```diff
@@ -411,7 +411,7 @@ def analyze_fn(match_group, fn, userspace_only):
     func, args = match_group
 
     try:
-        if args == "void":
+        if args == "void" or len(args) == 0:
```

Comment:
IIRC, compliance check would complain about empty argument list that has no `void` in it. If that's the case, I don't think we need this here.